README.md

```diff
@@ -1 +1,230 @@
+        path:"/"
+    }
+    gen_uuid(endpoint outboundEP, http:Request request) {
+        _ = outboundEP->respond(system:uuid());
+    }
+
+}
+```
+We will be building a docker image here, and publishing it to Docker Hub. This is required, since we cannot simply have the docker image in the local registry, and run the Kubernetes applicates in AKS, where it needs to have access to the docker image in a globally accessible location. For this, an image name should be given in the format $username/$image_name in the "image" property, and "username" and "password" properties needs to contain the Docker Hub account username and password respectively. The property "push" is set to "true" to signal the build process to push the build docker image to Docker Hub.
+
+You can build the Ballerina service using `$ ballerina build uuid_service.bal`. You should be able to see the following output. 
+
+```bash
+$ ballerina build uuid_service.bal
+Compiling source
+    uuid_service.bal
+
+Generating executable
+    uuid_service.balx
+	@kubernetes:Service 			 - complete 1/1
+	@kubernetes:Deployment 			 - complete 1/1
+	@kubernetes:Docker 			 - complete 3/3 
+	@kubernetes:Helm 			 - complete 1/1
+
+	Run the following command to deploy the Kubernetes artifacts: 
+	kubectl apply -f /home/laf/dev/uuid-service/kubernetes/
+
+	Run the following command to install the application using Helm: 
+	helm install --name uuid-service-deployment /home/laf/dev/uuid-service/kubernetes/uuid-service-deployment
+```
+
+After the build is done, the docker image would have been created, and pushed to Docker Hub, and also the Kubernetes deployment artifacts would be generated as well.
+    
+## Deployment
+
+- Configuring AKS environment
+
+Before deploying the service on AKS, you will need to setup the AKS environment to create the Kubernetes cluster and deploy an application.
+
+Let's start by installing the Azure command line utilities (CLI) in our local machine. Please refer to [Azure CLI Installation](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli?view=azure-cli-latest) in finding the steps for the installation.
+
+Next step is creating an Azure Service Principal:
+
+```bash 
+$ az ad sp create-for-rbac --skip-assignment
+```
+
+The output would be similar to the following:
+
+```bash 
+{
+  "appId": "e7596ae3-6864-4cb8-94fc-20164b1588a9",
+  "displayName": "azure-cli-2018-10-29-21-37-08",
+  "name": "http://azure-cli-2018-10-29-21-37-08",
+  "password": "52c95f25-bd1e-4314-bd31-d8112b293521",
+  "tenant": "72f988bf-86f1-41af-91ab-2d7cd011db48"
+}
+```
+
+- Create the Kubernetes cluster
+
+```bash 
+$ az aks create \
+    --resource-group myResourceGroup \
+    --name myAKSCluster \
+    --node-count 1 \
+    --service-principal <appId> \
+    --client-secret <password> \
+    --generate-ssh-keys
+```
+
+- Configure kubectl to connect to AKS Kubernetes cluster
+
+The following command will get the credentials for a given Kubernetes cluster in AKS.
```

Comment:
Change the tense to present. "The following command gets the credentials for a given Kubernetes cluster in AKS." 

Normally if you see the word "will" in a sentence it's an indicator of future tense. Avoid usage of this as the time an event occurs is uncertain to a user when we use future tense. Apply elsewhere if applicable.

Reply:
Got it, thanks!.